jep/200/README.adoc

```diff
@@ -67,19 +67,10 @@ Java’s built-in API, from the Remoting layer;
 and XStream, for persisting configuration and settings.
 Both permit any Java class to be loaded merely by specifying its name in the input.
 
-For years, the Jenkins project has received reports of remote code execution (RCE) attacks involving these frameworks.
```

Comment:
Diff too long to review; assuming you just rearranged content here.

Reply:
@jglick  Yes.